nb-adapters/adapter-dataapi/src/main/resources/activities/list_collection_names.yaml

```diff
@@ -6,4 +6,4 @@ blocks:
   list_collection_names:
     ops:
       op1:
-        list_collection_names:
+        list_collection_names: "TEMPLATE(collection1)"
```

Comment:
list_collection_names actually takes no args, so while you are correct that the yaml is broken without having *something* after the semicolon I don't think a templated var is appropriate, there's no need for the user to pass a value in. It could just be something hard coded there.

Reply:
@MarkWolters sounds good! What would you suggest is a good hard coded value to add in order to guide the user to fill the rest?

Reply:
@MikeYaacoubStax sorry for the delay. @jshook if you have an opinion we can go with that. To me it seems like you could just put the word "database" or something along those lines. I don't see an example of another op that takes no arguments. If we want to make it stand for something I would be ok with a TEMPLATE(database_name) or similar, I just don't think it makes sense to say list collections for a collection.

Reply:
For now, I would just require a value and not care what it is. There are a couple corner cases like this which we need to handle with a better yaml form, and this can be updated when we do that.

Reply:
All good, I just put in a hardcoded TBD